python/tank_vendor/sso_saml2/__init__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2015 Shotgun Software Inc.
```

Comment:
Your library has outdated copyrights btw.

Reply:
Do you know where I could get an updated copyrights header ?  For code and such...

Reply:
Just bump the year :)

Reply:
Pffff.... trop facile... pas de challenge. :-P